ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/monitor-telemetry-postcard-policy.j2

```diff
@@ -0,0 +1,65 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - monitor telemetry postcard policy #}
+{% if monitor_telemetry_postcard_policy is arista.avd.defined %}
+
+### Monitor Telemetry Postcard Policy
+{%     if monitor_telemetry_postcard_policy.sample_policies is arista.avd.defined %}
+
+#### Sample Policy Summary
+{%         for policy in monitor_telemetry_postcard_policy.sample_policies | arista.avd.natural_sort("name") %}
+
+##### {{ policy.name }}
+{%             if policy.match_rules is arista.avd.defined %}
+
+###### Match rules
+
+| Rule Name | Rule Type | Source Prefix | Destination Prefix | Protocol | Source Ports | Destination Ports |
+| --------- | --------- | ------------- | ------------------ | -------- | ------------ | ----------------- |
+{%                 for rule in policy.match_rules | arista.avd.natural_sort("name") %}
+{%                     set protocols_list = [] %}
+{%                     set destination_ports = [] %}
+{%                     set source_ports = [] %}
+{%                     for protocol in rule.protocols | arista.avd.natural_sort("protocol") %}
+{%                         do protocols_list.append(protocol.protocol) %}
+{%                         if protocol.source_ports is arista.avd.defined %}
+{%                             do source_ports.append(protocol.source_ports | join(", ")) %}
+{%                         else %}
+{%                             do source_ports.append("-") %}
+{%                         endif %}
+{%                         if protocol.destination_ports is arista.avd.defined %}
+{%                             do destination_ports.append(protocol.destination_ports | join(", ")) %}
+{%                         else %}
+{%                             do destination_ports.append("-") %}
+{%                         endif %}
+{%                     endfor %}
+{%                     if protocols_list == [] %}
+{%                         do protocols_list.append("-") %}
+{%                         do source_ports.append("-") %}
+{%                         do destination_ports.append("-") %}
+{%                     endif %}
+| {{ rule.name }} | {{ rule.type }} | {{ rule.source_prefix | arista.avd.default("-") }} | {{ rule.destination_prefix | arista.avd.default("-") }} | {{ protocols_list | join("<br>") }} | {{ source_ports | join("<br>") }} | {{ destination_ports | join("<br>") }} |
```

Comment:
Please join single word items with comma instead of <br>. It will make the tables a bit more compact and easier to read.

Reply:
We have some protocols as tcp and some as udp. To seperate these in a single row, added <br> between them.

Reply:
![Uploading Screenshot 2024-05-22 at 11.58.09 AM.png…]()

Reply:
OK, checked your test output again, and now I understand the idea. I will close the comments around this.